cfg/rke2-cis-1.24/master.yaml

```diff
@@ -335,6 +337,7 @@ groups:
           For example,
           chmod -R 600 /var/lib/rancher/rke2/server/tls/*.key
         scored: false
+        type: manual
```

Comment:
based on docs.rke2.io, 1.1.21 is automated: https://docs.rke2.io/security/cis_self_assessment124#1121-ensure-that-the-kubernetes-pki-key-file-permissions-are-set-to-600-automated

Do you think we should ignore it?